package.json

```diff
@@ -39,10 +36,14 @@
     "express": "4.13.4",
     "fh-mbaas-api": "5.14.2",
     "fh-wfm-mediator": "0.0.15",
-    "grunt": "^1.0.1",
-    "grunt-eslint": "^18.0.0",
+    "grunt": "0.4.5",
```

Comment:
In that case, it means grunt-mochify is not being maintained, and hasn't for quite some time, and I'd question whether it's wise to use it at all.